IO/GeoFileParser.h

```diff
@@ -26,32 +26,34 @@
 
 #include <tinyxml.h>
 
-//TODO: the class name GeoFileParser is misleading as the ``geometry'' file contains among others also relations (transitions)
-//TODO: between geometries/rooms. Probably, EnvironmentFileParser would be better, still parts of the environment are
-//TODO: currently stored in the ``ini'' file (e.g. goals) [gl march '16]
-//TODO: currently stored in the ``ini'' file (e.g. goals) [gl march '16]
-class GeoFileParser : public GeometryReader {
-
+// TODO: the class name GeoFileParser is misleading as the ``geometry'' file contains among others
+// also relations (transitions)
+// TODO: between geometries/rooms. Probably, EnvironmentFileParser would be better, still parts of
+// the environment are
+// TODO: currently stored in the ``ini'' file (e.g. goals) [gl march '16]
+// TODO: currently stored in the ``ini'' file (e.g. goals) [gl march '16]
+class GeoFileParser : public GeometryReader
+{
 public:
-     GeoFileParser(Configuration* configuration);
-     ~GeoFileParser();
-     virtual void LoadBuilding(Building* building) override;
-
-     virtual bool LoadTrafficInfo(Building* building) override;
-     bool parseDoorNode(TiXmlElement * xDoor, int id, Building* building);
-     Goal* parseGoalNode(TiXmlElement * e);
-     Transition* parseTransitionNode(TiXmlElement * xTrans, Building * building);
-     Goal* parseWaitingAreaNode(TiXmlElement * e);
-     bool LoadTrainInfo(Building* building);
-     bool LoadTrainTimetable(Building* building, TiXmlElement * xRootNode);
-     bool LoadTrainType(Building* building, TiXmlElement * xRootNode);
-     std::shared_ptr<TrainType> parseTrainTypeNode(TiXmlElement * e);
-     std::shared_ptr<TrainTimeTable> parseTrainTimeTableNode(TiXmlElement * e);
+    GeoFileParser(Configuration * configuration);
+    ~GeoFileParser();
+    virtual void LoadBuilding(Building * building) override;
+
+    virtual bool               LoadTrafficInfo(Building * building) override;
+    bool                       parseDoorNode(TiXmlElement * xDoor, int id, Building * building);
+    Goal *                     parseGoalNode(TiXmlElement * e);
+    Transition *               parseTransitionNode(TiXmlElement * xTrans, Building * building);
+    Goal *                     parseWaitingAreaNode(TiXmlElement * e);
+    bool                       LoadTrainInfo(Building * building);
+    bool                       LoadTrainTimetable(Building * building, TiXmlElement * xRootNode);
+    bool                       LoadTrainType(Building * building, TiXmlElement * xRootNode);
+    std::shared_ptr<TrainType> parseTrainTypeNode(TiXmlElement * e);
+    std::shared_ptr<TrainTimeTable> parseTrainTimeTableNode(TiXmlElement * e);
```

Comment:
What's the logic here? How come we have so many spaces between the type and the name of the function?

Reply:
@chraibi the current clang-format config is set to align blocks of declarations so that names start at the same column. I am not such a fan of that either :)

Reply:
Not minimalistic, but ok.